cdap-master/src/main/java/co/cask/cdap/master/startup/ConfigurationCheck.java

```diff
@@ -100,6 +105,32 @@ private void checkServiceResources(Set<String> problemKeys) {
     }
   }
 
+  private void checkBindAddresses() {
+    // check if service bind addresses are loopback addresses
+    ImmutableSet.Builder<String> builder = ImmutableSet.<String>builder()
```

Comment:
think you can only use that for 6 items or less...
However you can save a line of code by not using a variable for the builder, since you're not using the builder after line 119. Just have this be a ``Set<String>`` and call ``.build()`` on line 119?